packages/browser-wallet/src/popup/pages/AddWeb3IdCredential/AddWeb3IdCredential.tsx

```diff
@@ -167,27 +172,56 @@ export default function AddWeb3IdCredential({ onAllow, onReject }: Props) {
         [metadata, i18n]
     );
 
+    async function findNextUnusedVerifiableCredentialIndex(
```

Comment:
If you add the client as an argument you could move this out of the component.

Reply:
Good idea. I have done this.